### ISSUE TITLE
[Task]: Implement SimpleNet Paper

### ISSUE THREAD

**Issue by willyfh** [openvinotoolkit/anomalib#1603]
### What is the motivation for this task?

Implementation of SimpleNet , one of the sota anomaly detection models published in CVPR 2023.

### Describe the solution you'd like

📄 [Paper](https://openaccess.thecvf.com/content/CVPR2023/papers/Liu_SimpleNet_A_Simple_Network_for_Image_Anomaly_Detection_and_Localization_CVPR_2023_paper.pdf)
🧑‍💻 [Code](https://github.com/DonaldRR/SimpleNet)

**Comment by samet-akcay** [comment]
I agree that we should have this in anomalib. Let's convert this to a feature request
